openhtmltopdf-core/src/main/java/com/openhtmltopdf/extend/Hyphenator.java

```diff
@@ -3,6 +3,7 @@
 /**
  * Interface for a custom hyphenation implementation.
  */
+@FunctionalInterface
 public interface Hyphenator {
     String hyphenateText(String text);
```

Comment:
Could you add a JavaDoc comment onto `hyphenateText` here which explains that this function is expected to return the input string transformed to include soft hyphens where allowed?